SECURITY.md

```diff
@@ -6,39 +6,19 @@ The following table describes the versions of this project that are currently su
 
 | Version | Supported          |
 | ------- | ------------------ |
-| 3.x   | :white_check_mark:   |
-| 4.x   | :white_check_mark:   |
-| 5.x   | :white_check_mark:   |
+| 3.x   | :heavy_check_mark:   |
+| 4.x   | :heavy_check_mark:   |
+| 5.x   | :heavy_check_mark:   |
 
-## Responsible disclosure security policy
+## Responsible disclosure policy
 
-A responsible disclosure policy helps protect users of the project from publicly disclosed security vulnerabilities
-without a fix by employing a process where vulnerabilities are first triaged in a private manner, and only publicly
-disclosed after a reasonable time period that allows patching the vulnerability and provides an upgrade path for users.
+At Sequelize, we prioritize security issues and will try to fix them as soon as they are disclosed.
 
-When contacting us directly via email, we will do our best efforts to respond in a reasonable time to resolve the issue.
-When contacting a security program their disclosure policy will provide details on timeframe, processes and paid bounties.
+If you discover a security vulnerability, please reach the project maintainers privately. You can find related information in [CONTACT.md](./CONTACT.md).
```

Comment:
@sushantdhiman maybe "reach out to the...." instead of "reach the" ?

Reply:
de06ac3fd714f7c7eeb10db5774724007928f0c1